### PR TITLE
API Changes: Sol-platform

### PR DESCRIPTION
**Pull request by amandawrcoelho**
Move CHUNK_READ_SIZE, CHUNK_READ_MAX and CHUNK_MAX_TIME_NS to
sol-util-internal.h

Rename function

Now sol_platform_get_state() return an enum

issue #1675

Signed-off-by: Amanda Coelho <amanda.winder.ribeiro.coelho@intel.com>

### CODE REVIEW

**Comment by ceolin** [comment]
looks ok

File: src/lib/common/sol-platform.c [hunk 1]
```diff
@@ -206,7 +207,7 @@ sol_platform_get_board_name(void)
     return board_name;
 }
 
-SOL_API int
+SOL_API enum sol_platform_state
 sol_platform_get_state(void)
 {
     return sol_platform_impl_get_state();
```
Comment (otaviobp) [inline src/lib/common/sol-platform.c]: I would change the return type from sol_platform_impl_get_state to enum too.

Comment (otaviobp) [inline src/lib/common/sol-platform.c]: And you will need to fix the implementation of contiki and zephyr.

Comment (amandawrcoelho) [inline src/lib/common/sol-platform.c]: After a brief talk, its better dont change the type of this function, because enum type its not the best way.

**Pull request closed without merging**